app/src/test/java/org/mozilla/focus/settings/SearchEngineValidationTest.kt

```diff
@@ -24,9 +24,9 @@ class SearchEngineValidationTest {
     }
 
     @Test
-    fun `URL using HTTP redirect is valid`() = withMockWebServer(responseWithStatus(301)) {
-        // Currently we accept redirects as valid. We might want to follow the redirect (Issue #1976)
-        assertTrue(isValidSearchQueryURL(it.rootUrl()))
+    fun `URL using HTTP redirect is invalid`() = withMockWebServer(responseWithStatus(301)) {
+        // We now follow redirects(Issue #1976). This test now asserts false.
```

Comment:
nit: similar to my comment above, this comment references the past state of the code, "We **now** follow redirects... This test **now** asserts...". I think a better description might be: "We follow redirects so we should never receive a 3xx. If we do, we want to fail." Or something.

But I don't think it's worth the turn around time to fix now.

Reply:
Will keep in mind the next time 😄 Thanks for helping me through!